docs/2-configuration.md

```diff
@@ -44,6 +44,7 @@ Here's what you can configure:
 | `WG_WIREGUARD_INTERFACE`             | `--wireguard-interface`             | `wireguard.interface`          |          | `wg0`                                        | The wireguard network interface name                                                                                                                                                                                                                                          |
 | `WG_WIREGUARD_PRIVATE_KEY`           | `--wireguard-private-key`           | `wireguard.privateKey`         | Yes      |                                              | The wireguard private key. This value is required and must be stable. If this value changes all devices must re-register.                                                                                                                                                     |
 | `WG_WIREGUARD_PORT`                  | `--wireguard-port`                  | `wireguard.port`               |          | `51820`                                      | The wireguard server port (udp)                                                                                                                                                                                                                                               |
+| `WG_WIREGUARD_MTU`                   | `--wireguard-mtu`                   | `wireguard.mtu`                |          |                                              | The maximum transmission unit (MTU) used on the service-side interface.                                                                                                                                                                                                       |
```

Comment:
Maybe we can add a hint what happens if left empty to both MTU items, e.g. "If left empty a sensible default is used".